api/auth.go

```diff
@@ -70,6 +72,7 @@ func WithToken(token string) Option {
 	return clientFunc(func(c *Client) error {
 		c.log.Debug("setting up auth", zap.String("token", token))
 		c.auth.token = token
+		c.auth.expiresAt = time.Now().Add(defaultTimeout)
```

Comment:
Im not sure this is the right default token expiry time.... there should be a different variable...

Reply:
Found it! We should use `DefaultTokenExpiryTime ` instead.